packages/components/src/components/button/button-styles.ts

```diff
@@ -64,6 +64,7 @@ export const getComponentCss = (
           }),
         }),
         ...(loading && !isPrimary && frostedGlassStyle),
+        margin: 0, // Removes default button margin on safari 15
```

Comment:
Could be reordered to be next to other layout styles.

Reply:
There are no other layout styles within the `root` selector?